### ISSUE TITLE
tst_30 fails

### ISSUE THREAD

**Issue by hokru** [qrefine/qrefine#47]
**Describe the bug**
The lbfgs optimization fails with:

```
micro_cycle  13 rmsd(b):  0.0003 rws:  1.000 n_fev: 700
  step:   1 bond rmsd: 0.000327
  step:   2 bond rmsd: 0.128277
  step:   3 bond rmsd: 0.000326
  step:   4 bond rmsd: 0.000327
  step:   5 bond rmsd: 0.000327
minimization failed: lbfgs error: Line search failed: The step is at the upper bound stpmax().
```

Unclear what was changed in the past that causes this.

**Comment by pafonine** [comment]
All but one tests run for me (using latest code). The one that crashes is

qr.refine 
/Users/pafonine/phenix/modules/qrefine/tests/unit/data_files/m00_poor.pdb 
restraints=cctbx mode=opt use_convergence_test=False 
output_folder_name=tst_30 > tst_30.log
tst_30 FAILED Cannot open file for reading: "tst_30/m00_poor_refined.pdb"
   [Errno 2] No such file or directory: 'tst_30/m00_poor_refined.pdb'
        Total time: 1245.79 (s)
        Total time: 1246.43 (s)
        Total time: 1247.09 (s)
Pyscf not installed. Please run: phenix.python pip -m  install pyscf
tst_33: Skipped (not recommended, enable ASAP or remove!)
        Total time: 1247.73 (s)
        Total time: 1248.36 (s)
qr.refine 
/Users/pafonine/phenix/modules/qrefine/tests/unit/data_files/m00_poor.pdb 
restraints=cctbx mode=gtest g_scan=20 g_mode=1 output_folder_name=tst_35 
 > tst_35.log
tst_35:  OK  Time:  15.93 (s)
        Total time: 1264.91 (s)
Failed tests : 4
Traceback (most recent call last):
   File 
"/Users/pafonine/phenix/build/../modules/qrefine/command_line/test.py", 
line 59, in <module>
     assert not rc, 'qr.test rc : %s' % rc
AssertionError: qr.test rc : 1
pva_mbp:/Users/pafonine/phenix/rt %

Pavel

**Comment by pafonine** [comment]
OK, I think I fixed it.
Pavel

**Comment by hokru** [comment]
confirmed fixed with 09069e6f8086c3542f6468ac66da52978e48f985
